injectors/linux.js

```diff
@@ -7,33 +7,62 @@ const { BasicMessages, AnsiEscapes } = require('./log');
 // This is to ensure the homedir we get is the actual user's homedir instead of root's homedir
 const homedir = execSync('grep $(logname) /etc/passwd | cut -d ":" -f6').toString().trim();
 
-const KnownLinuxPaths = Object.freeze([
-  '/usr/share/discord-canary',
-  '/usr/lib64/discord-canary',
-  '/opt/discord-canary',
-  '/opt/DiscordCanary',
-  `${homedir}/.local/bin/DiscordCanary` // https://github.com/powercord-org/powercord/pull/370
-]);
+const KnownLinuxPaths = {
+  stable: Object.freeze([
+    '/usr/share/discord',
+    '/usr/lib64/discord',
+    '/opt/discord',
+    '/opt/Discord',
+    `${homedir}/.local/bin/Discord`
+  ]),
+  ptb: Object.freeze([
+    '/usr/share/discord-ptb',
+    '/usr/lib64/discord-ptb',
+    '/opt/discord-ptb',
+    '/opt/DiscordPTB',
+    `${homedir}/.local/bin/DiscordPTB`
+  ]),
+  canary: Object.freeze([
+    '/usr/share/discord-canary',
+    '/usr/lib64/discord-canary',
+    '/opt/discord-canary',
+    '/opt/DiscordCanary',
+    `${homedir}/.local/bin/DiscordCanary` // https://github.com/powercord-org/powercord/pull/370
+  ]),
+  dev: Object.freeze([
+    '/usr/share/discord-development',
+    '/usr/lib64/discord-development',
+    '/opt/discord-development',
+    '/opt/DiscordDevelopment',
+    `${homedir}/.local/bin/DiscordDevelopment`
+  ])
+};
```

Comment:
freeze the entire object (`const KnownLinuxPaths = Object.freeze({ stable: [ ... ], ... })`) instead of each of the arrays